lib/pages/artist_page.dart

```diff
@@ -43,63 +40,65 @@ class ArtistPage extends StatelessWidget {
-                    : null,
-                artist.website != null
-                    ? new ListTile(
-                        leading: const Icon(FontAwesomeIcons.chrome),
-                        title: new Text(artist.websiteShort),
-                        onTap: () => launch(artist.website),
-                      )
-                    : null,
-                new Divider(height: 32.0),
-                new Column(
-                  crossAxisAlignment: CrossAxisAlignment.start,
-                  children: <Widget>[
-                    new ListTile(
-                      leading: const Icon(Icons.info),
-                      title: new Text(
-                        'Biographie',
-                        style: Theme.of(context).textTheme.subhead,
-                      ),
-                    ),
-                    new Container(
-                      padding: new EdgeInsets.only(
-                        left: 72.0,
-                        right: 16.0,
-                        bottom: 16.0,
-                      ),
-                      child: new Text(artist.bio,
-                          style: Theme.of(context).textTheme.body1),
-                    ),
-                  ],
-                )
-              ]..removeWhere((Widget w) => w == null),
-            ),
+            delegate:
+                new SliverChildListDelegate(socialWidgets(artist, context)),
           ),
         ],
       ),
     );
   }
 }
+
+Column bioWidgets(BuildContext context, Artist artist) => new Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: <Widget>[
+        new ListTile(
+          leading: const Icon(Icons.info),
+          title: new Text(
+            'Biographie',
+            style: Theme.of(context).textTheme.subhead,
+          ),
+        ),
+        new Container(
+          padding: new EdgeInsets.only(
+            left: 72.0,
+            right: 16.0,
+            bottom: 16.0,
+          ),
+          child: new Text(artist.bio, style: Theme.of(context).textTheme.body1),
+        ),
+      ],
+    );
+
+List<Widget> socialWidgets(Artist artist, BuildContext context) => [
+      artist.facebook != null
+          ? new ListTile(
+              leading: const Icon(FontAwesomeIcons.facebook),
+              title: new Text(artist.facebookHandle),
+              onTap: () => launch(artist.facebook),
+            )
+          : null,
+      artist.twitter != null
+          ? new ListTile(
+              leading: const Icon(FontAwesomeIcons.twitter),
+              title: new Text(artist.twitterHandle),
+              onTap: () => launch(artist.twitter),
+            )
+          : null,
+      artist.youtube != null
+          ? new ListTile(
+              leading: const Icon(FontAwesomeIcons.youtube),
+              title: new Text(artist.youtubeHandle),
+              onTap: () => launch(artist.youtube),
+            )
+          : null,
+      artist.website != null
+          ? new ListTile(
```

Comment:
This `ListTile` looks like a reusable component to me. 😛

Reply:
@emileber there you go mah man